.storybook/themes/news.scss

```
@@ -685,6 +685,41 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					background: var(--background-color),
+					box-shadow: var(--global-box-shadow-1),
+					border-radius: 0 var(--border-radius) var(--border-radius) 0,
+					display: none,
+					left: 0,
+					padding: var(--global-spacing-4) 0,
+					position: fixed,
+					top: 200px,
+					width: 44px,
+					components: (
+						button-medium: (
+							background: transparent,
+							border-width: 0,
+							padding: 0 10px,
+							text-align: center,
+							width: 44px,
+						),
+					),
+				),
```

We should not be setting explicit `width` values if we don't need to, this allows the contents to grow responsively 

```suggestion
				share-bar: (
					background: var(--background-color),
					box-shadow: var(--global-box-shadow-1),
					border-radius: 0 var(--border-radius) var(--border-radius) 0,
					display: none,
					gap: var(--global-spacing-2),
					left: 0,
					padding: var(--global-spacing-4) 0,
					position: fixed,
					top: 200px,
					components: (
						button-medium: (
							background: transparent,
							border-width: 0,
							padding: 0 var(--global-spacing-3),
						),
					),
				),
```